Formula/feroxbuster.rb

```diff
@@ -0,0 +1,32 @@
+class Feroxbuster < Formula
+  desc "Fast, simple, recursive content discovery tool written in Rust"
+  homepage "https://epi052.github.io/feroxbuster"
+  url "https://github.com/epi052/feroxbuster/archive/v2.3.3.tar.gz"
+  sha256 "5a244d1614fb9dc476e9be8bd1cdbb33e0b07a1fcb7285e911ae6c77a44c7327"
+  license "MIT"
+
+  depends_on "rust" => :build
+  depends_on "miniserve" => :test
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    (testpath/"wordlist").write <<~EOS
+      a.txt
+      b.txt
+    EOS
+
+    (testpath/"web").mkpath
+    (testpath/"web/a.txt").write "a"
+    (testpath/"web/b.txt").write "b"
+
+    fork do
+      exec "miniserve", testpath/"web"
+    end
```

Comment:
We should probably make sure that the server gets shut down at the end of the test. See `miniserve`'s own test for how to do this.

Reply:
Thanks, I updated it properly shut down `miniserve`, same as in its own tests.